### ISSUE TITLE
Structured data improvements

### ISSUE THREAD

**Issue by zabi332** [ahmedkaludi/accelerated-mobile-pages#1433]

**Comment by Frenchomatic** [comment]
1. For a static home page it is adding the site title " | mysitetitle " into the headline and not just the page title which happens on every other type of page. I don't think the home page should be different. 

2.  It would be very useful to have an admin entry area to put json or hreflangs into the header on a page per page basis. For example, including local business markup within an AMP page. Genesis users already get this on canonical pages whilst others use plugins.

**Comment by ahmedkaludi** [comment]
3. A blog posts page, category or archive containing posts should have structured markup as @type Blog and contain all of the individual posts which are @type blogposting. This perhaps needs to be a setting in admin area.

As in previous forum post – if the home page is static (i.e. is not a blog posts page even if one exists on the site) it needs to be just a Webpage.

These should be independent of each other as well. In other words, a static home page gets Webpage and if a blog posts page exists then it is @type Blog.

https://wordpress.org/support/topic/structured-data-blog-posts-page-etc/

**Comment by ahmedkaludi** [comment]
Can we test this please? 

and also i'd like your feedback on this @Frenchomatic

**Comment by Frenchomatic** [comment]
sure - where are the files

**Comment by Frenchomatic** [comment]
Yes but it it is a feature. If you look at a standard genesis set up for a blog posts page you will see that the structured data is different. More important is to get the 404 pages for AMP takeover to work again - it is causing big problems.

**Comment by Zabi09** [comment]
@Frenchomatic 
Will you please give us an URL of any of the blog posts of your Genesis theme where the structured data is different So that we will check once and let you know.

**Comment by Zabi09** [comment]
And I have also checked it and SEO description and structure data are appearing fine in both Genesis and in the amp version. Please provide us URL and we will check and let you know.

**Comment by Frenchomatic** [comment]
If you look at a standard genesis set up for a blog posts page you will see that the structured data is different.  The AMP plugin doesn't include the creativeworks. Take this page for example - https://demo.studiopress.com/genesis-sample/
